test.sh

```diff
@@ -1,21 +1,27 @@
 #!/usr/bin/env bash
 
-function finish {
-  echo '-----------------------test.sh------------------------------'
-  echo 'Removing test environment'
-  echo '------------------------------------------------------------'
-  docker-compose down -v
+function main() {
```

Comment:
You should always try a cleanup first before starting anything otherwise you can't be sure that your environment is properly cleaned.

Reply:
before each run, I do run this command. I just wanted to comment it out for troubleshooting purposes. To drop down into the containers if needed to check env variables, etc. I do plan to uncomment this out before the final push

Reply:
I still don't see cleanup _before_ the start of the tests